### DISCUSSION TITLE
在内网里使用的时候出现问题。

### DISCUSSION THREAD

**Discussion by aymayi** [hslr-s/sun-panel#198] [General]
内网部署完成后。360浏览器无法正常浏览。

**Comment by amwcel827** [comment]
我也遇到了。但是用edge浏览器能正常打开。这样就造成使用范围受限了。

**Comment by aymayi** [comment]
不知道设置哪里可以解决。谷歌浏览器也可以，但是内网都是使用360浏览器，希望能早日解决。

**Comment by hslr-s** [comment]
将360浏览器升级到最新版本试试

**Comment by amwcel827** [comment]
现在麻烦的就是这点。我们内部需要使用低版本的。





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "hslr-s/sun-panel"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月16日(星期二) 上午10:33
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [hslr-s/sun-panel] 在内网里使用的时候出现问题。 (Discussion #198)





 
将360浏览器升级到最新版本试试
